virttest/utils_conn.py

```diff
@@ -1292,17 +1300,18 @@ def server_setup(self, on_local=False):
             raise ConnMkdirError(self.libvirt_pki_private_dir, output)
 
         if scp_new_cacert == 'no':
-            scp_dict = {servercert_path: self.libvirt_pki_dir,
-                        serverkey_path: self.libvirt_pki_private_dir}
+            params_scp = {'serverkey': serverkey_path,
+                          'servercert': servercert_path}
         else:
-            scp_dict = {cacert_path: self.pki_CA_dir,
-                        cakey_path: self.pki_CA_dir,
-                        servercert_path: self.libvirt_pki_dir,
-                        serverkey_path: self.libvirt_pki_private_dir}
             cmd = "mkdir -p {0}".format(self.pki_CA_dir)
             status, output = server_session.cmd_status_output(cmd)
             if status:
                 raise ConnMkdirError(self.pki_CA_dir, output)
+            params_scp = {'cacert': cacert_path,
```

Comment:
Here what params_scp makes difference with params_scp in line 1303  are newly added two keys:cacert,cakey. 
So the proper way could be using params_scp.update(xx)

Reply:
This is just to define a new dict variable (params_scp) with different initial values. params_scp is not defined before this statement. So I think params_scp.update() is not proper.